packages/protocol/scripts/bash/release-lib.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+function build_tag() {
+  BRANCH="$1"
+  LOG_FILE="$2"
+
+  echo " - Checkout contracts source code at $BRANCH"
+  BUILD_DIR=$(echo build/$(echo $BRANCH | sed -e 's/\//_/g'))
+  git checkout --no-overlay $BRANCH -- contracts 2>>$LOG_FILE >> $LOG_FILE
+
+  echo " - Build contract artifacts at $BUILD_DIR"
+  mv build/contracts build/contracts_tmp
+  yarn build:sol >> $LOG_FILE
```

Comment:
Are we sure that this will work? I already lost track of all the issues we have faced so far and it might have been during tooling building instead